modules/gapi/include/opencv2/gapi/video.hpp

```diff
@@ -169,8 +330,58 @@ calcOpticalFlowPyrLK(const GArray<GMat>    &prevPyr,
                            int              flags        = 0,
                            double           minEigThresh = 1e-4);
 
+/** @brief Gaussian Mixture-based or K-nearest neighbours-based Background/Foreground Segmentation Algorithm.
+The operation generates a foreground mask.
+
+@return Output image is foreground mask, i.e. 8-bit unsigned 1-channel (binary) matrix @ref CV_8UC1.
+
+@note Functional textual ID is "org.opencv.video.BackgroundSubtractor"
+
+@param src input image: Floating point frame is used without scaling and should be in range [0,255].
+@param bsParams Set of initialization parameters for Background Subtractor kernel.
+*/
+GAPI_EXPORTS GMat BackgroundSubtractor(const GMat& src, const cv::gapi::video::BackgroundSubtractorParams& bsParams);
+
+/** @brief Standard Kalman filter <http://en.wikipedia.org/wiki/Kalman_filter>.
+However, you can modify transitionMatrix, controlMatrix,
+and measurementMatrix to get an extended Kalman filter functionality.
+
+Output image is predicted or corrected state, i.e. 32-bit or 64-bit float 1-channel
+matrix @ref CV_32F or CV_64F.
+Predicted state will be returned, if measurement matrix is empty.
+And vice versa corrected state will be returned, if measurement matrix is empty.
```

Comment:
This description is not correct. Since If measurement matrix is given (haveMeasurements == true) corrected state will be returned. And otherwise, predicted state will be returned which corresponds to the call of cv::KalmanFilter::predict(control).

Reply:
Oh, right, sorry for that. Of course, I mismatched the cases.
The main idea was to correct and widen the description a little bit, if you like, explicitly say what the kernel does and what user should expect.
Here is what I wanted to write:
```
If measurement matrix is not given (haveMeasurements == false), predicted state will be returned which corresponds to the call of cv::KalmanFilter::predict(control). Otherwise, corrected state will be returned which corresponds to the pipeline cv::KalmanFilter::predict(control) -> cv::KalmanFilter::correct(measurement).
```

Reply:
Applied.